bluepysnap/nodes.py

```diff
@@ -545,14 +545,18 @@ def get(self, group=None, properties=None):
                 - ``mapping``: return the properties of nodes matching a properties filter.
                 - ``None``: return the properties of all nodes.
 
-            properties (list): If specified, return only the properties in the list.
+            properties (list|str): If specified, return only the properties in the list.
                 Otherwise return all properties.
 
         Returns:
             value/pandas.Series/pandas.DataFrame:
-                If single node ID is passed as ``group`` and single property as properties returns
-                a single value. If single node ID is passed as ``group`` and list as property
-                returns a pandas Series. Otherwise return a pandas DataFrame indexed by node IDs.
+                If a single node ID is passed as ``group`` and a single property as ``properties``,
+                returns a single value.
+                If a single node ID is passed as ``group`` and a list as ``properties``,
+                returns a pandas Series.
+                If something different from a single node ID is passed as ``group`` and a single
+                property as ``properties``, returns a pandas Series.
```

Comment:
I read this sentence as:
`len(group) !=1 AND len(properties) != 1`
while I believe the meaning behind the sentence is:
`len(group) !=1 AND len(properties) == 1`

Maybe 
"...single node ID is passed as ``group`` and a single property is passed as ``properties``..."
would do the trick?

I don't know. I'm not a native speaker either so... 🤷‍♂️ 

Also, I'd use: `If anything else than a single node ID...` but I'm still not a native speaker 🤣

Reply:
Yeah, I'll rephrase it. Note that if you pass `1` instead of `[1]` as `group` the results are different, so I can add an example for each case to make it clearer.